File: Assignments/Assignment_12/p3.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                int count=i;
                for(int j=0;j<row;j++){
                        printf("%d\t",(count));
                        count++;

                }
                printf("\n\n");
        }
}

File: JAVA/12.Space_Triangle/p3.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows:");
		int row=sc.nextInt();

		int space=row-2;

		//char count =(char)(64+row);
		for(int i=0;i<row;i++){
		
			char count=(char)(64+row-i);
			for(int j=0;j<row;j++){
			
				if(j<=space){
				
					System.out.print("\t");
				}else{
				
					System.out.print(count+"\t");
					count++;
				}
				
			}space--;
			
			System.out.println();
		}
	}
}
